plugins/woocommerce/src/Admin/Features/Settings/Transformer.php

```diff
@@ -0,0 +1,323 @@
+<?php
+/**
+ * WooCommerce Settings Data Transformer.
+ */
+
+declare( strict_types = 1 );
+
+namespace Automattic\WooCommerce\Admin\Features\Settings;
+
+/**
+ * Transforms WooCommerce settings data into a structured format with logical groupings.
+ */
+class Transformer {
```

Comment:
Thanks for the suggestion! I'm not very sure about this because I think most of code is transforming settings. Tab and Section are just simple logic (two methods). `SettingTransformer` will still be a big class. 🤔

Reply:
I've converted this from static to instance-based. dee9986cd0f5a61b57febd9070750ed77b41cacf

Reply:
> I'm not very sure about this because I think most of code is transforming settings. Tab and Section are just simple logic (two methods). SettingTransformer will still be a big class. 🤔

Got it, yeah I guess separating doesn't add much value at this point. Sounds good to keep it!